### PR TITLE
Adding Portuguese Translation

### PR DESCRIPTION
**Pull request by Fausto95**
Adding PT translations 🇦🇴🇧🇷🇵🇹

### CODE REVIEW

**Comment by lipis** [comment]
Soon we are going to need a different UI for all those languages.. this UI doesn't scale well :)

File: src/i18n.ts [hunk 1]
```diff
@@ -17,7 +17,8 @@ export function parseDetectedLang(lng: string | undefined): string {
 export const languages = [
   { lng: "en", label: "English" },
   { lng: "es", label: "Español" },
-  { lng: "fr", label: "Français" }
+  { lng: "fr", label: "Français" },
+  { lng: "pt", label: "Português" },
```
Comment (lipis) [inline src/i18n.ts]:
```suggestion
  { lng: "pt", label: "Português" }
```

to pass the lint

Comment (Fausto95) [reply src/i18n.ts]: Yes

**Comment by Fausto95** [comment]
Merging this right now and I will be enhancing the language selection tomorrow!

**Comment by lipis** [comment]
Do you mind updating the translations on Crowdin for Portuguese?

https://crowdin.com/project/excalidraw

**Pull request merged**